recipe/meta.yaml

```
@@ -19,7 +19,7 @@ build:
   # Fixing requires to rebuild the whole stack while OSX build
   # for python 3.8 is not used  by anyone.
   skip: true  # [osx and py<39]
```

This error for osx_arm64 is on me.

For now we can skip with:
```suggestion
  skip: true  # [osx and py<39]
  # TODO: publish unordered_dense for `osx_arm64` on conda-forge
  skip: true  # [osx and x86_64]
```

If you apply this suggestion, we will need to rebuild.